edx_dl/edx_dl.py

```diff
@@ -326,7 +365,8 @@ def extract_units(url, headers):
     """
     Parses a webpage and extracts its resources e.g. video_url, sub_url, etc.
     """
-    compat_print("Processing '%s'" % url)
+    logging.info("Processing '%s'", url)
```

Comment:
I really think we must change this to be part of the debug information in a future commit. However I don't know how to show the progress to the user in a user friendly way (e.g. a progress bar or text '## / TOTAL')